pages/governance/index.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div>
+    <div
+      v-if="
+        proposals.length === 0 || Object.keys(governanceOverview).length === 0
+      "
+      class="loading-row"
+    >
+      Loading...
+    </div>
+
+    <Card v-else-if="!proposals.length">
+      <div slot="title">No proposals</div>
+      <!-- TODO: add some actionable button / link -->
+      <div slot="subtitle">Submit now your own proposal to this blockchain</div>
+    </Card>
+
+    <template v-else>
+      <PageProposals
+        class="page"
+        :proposals="proposals"
+        :governance-overview="governanceOverview"
+      />
+
+      <template>
+        <p class="message">
+          {{ oldChainDataMessage }}
+        </p>
+      </template>
+    </template>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+
+export default {
+  name: `proposals-index`,
+  computed: {
+    ...mapState('data', [`proposals`, `governanceOverview`]),
+    ...mapState(['session']),
+    oldChainDataMessage() {
+      return `If you expected to see proposals here that are missing, 
```

Comment:
This is not a thing for proposals

Reply:
But it happens the same for proposals, no? Or is that not the case?? :eyes: Are the cross-blockchain-version persistent?